sbin/pfdhcplistener

```diff
@@ -107,6 +109,8 @@ sub reload_config {
     }
     elsif ( pf::cluster::is_vip_running($interface) ) { 
         $process = $TRUE;
+        $is_clustered = $TRUE;
+        $holds_vip = $TRUE;
```

Comment:
You can get the VIP back when you pry it from my cold, dead hands...

All right, I'll give it back...
Fixing this.